ADLStream/data/preprocessing/standardization_scaler.py

```diff
@@ -0,0 +1,132 @@
+"""Implements a standardization scaler"""
+from ADLStream.data.preprocessing import BasePreprocessor
+import math
+
+
+class StandardizationScaler(BasePreprocessor):
+    """The transformation is given by
+        x_scaled = (x - avg_x) / x_stdev
+
+    where avg_x is the mean seen until now for the feature x and x_stdev
+    is the standard deviation seen until now for the feature x.
+
+    Arguments:
+        share_params (bool): Whether to share scaler parameters among columns.
+            Defaults to False.
+
+    """
+
+    def __init__(self, share_params=False):
+        self.share_params = share_params
+        self.data = None
+        self.data_sum = None
+        self.data_stdev_sum = None
+        self.data_avg = None
+        self.data_stdev = None
+        self.data_count = 1
+
+    def _mean(self, a):
+
+        if self.share_params == False:
+            assert len(a) == len(self.data_sum)
+            self.data_sum = [self.data_sum[i] + a[i] for i in range(len(a))]
+            mean = [(self.data_sum[i]) / self.data_count for i in range(len(a))]
+
+        else:
+            self.data_sum += sum(a)
+            mean = [self.data_sum / (self.data_count * len(a))] * len(a)
+
+        return mean
+
+    def _standard_deviation(self, a):
```

Comment:
This looks great. 
As discussed, can you add the docstring referencing the `std` formula used?